Resources/config/commands.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="Sonata\AdminBundle\Command\CreateClassCacheCommand" class="Sonata\AdminBundle\Command\CreateClassCacheCommand">
```

Comment:
If the `id` is a FQCN you don't need to define `class` keyword right? Why are we adding both?

Reply:
Interesting... do you have a source to back that up? Would it work on all the versions of sf we are compatible with?

Reply:
@jordisala1991 See https://github.com/symfony/symfony/blob/2.8/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/AddConsoleCommandPass.php#L39
`class` option is required for command

Reply:
on `2.8` is required yes, then I didn't say anything :)